input/pagecontent/index.md

```diff
@@ -17,6 +17,25 @@ The AU eRequesting project includes:
 
 {% include dependency-table.xhtml %}
 
+### AU eRequesting Actors
+
+The following actors are part of AU eRequesting:
+
+**AU eRequesting Placer**
+
+A system is responsible for initiating and creating patient diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that all required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
+
+
+**AU eRequesting Filler**
+
+A system responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them It ensures that service providers have the necessary details to fulfil the requests and manages the requests through the fulfilment process.
```

Comment:
Missing fullstop between "them" and "It".

Reply:
Addressed via PR #39